src/Neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -338,12 +338,13 @@ public JObject ToJson(ProtocolSettings settings)
         /// <param name="settings">The <see cref="ProtocolSettings"/> used to verify the transaction.</param>
         /// <param name="snapshot">The snapshot used to verify the transaction.</param>
         /// <param name="context">The <see cref="TransactionVerificationContext"/> used to verify the transaction.</param>
+        /// <param name="conflictsList">The list of conflicting <see cref="Transaction"/> those fee should be excluded from sender's overall fee during <see cref="TransactionVerificationContext"/>-based verification in case of sender's match.</param>
```

Comment:
I think it needs rewrite. `those` is strange.

Reply:
> those is strange.

Could you add a suggestion? How about the following:
```
The list of conflicting <see cref="Transaction"/>. Fee of these transactions should be excluded from sender's overall fee during <see cref="TransactionVerificationContext"/>-based verification in case of sender's match.</param>
```